x-pack/test/api_integration/apis/synthetics/get_monitor.ts

```diff
@@ -48,23 +48,23 @@ export default function ({ getService }: FtrProviderContext) {
 
     describe('get many monitors', () => {
       it('without params', async () => {
-        const [{ id: id1, attributes: mon1 }, { id: id2, attributes: mon2 }] = await Promise.all(
-          monitors.map(saveMonitor)
-        );
+        const [mon1, mon2] = await Promise.all(monitors.map(saveMonitor));
 
         const apiResponse = await supertest
           .get(API_URLS.SYNTHETICS_MONITORS + '?perPage=1000') // 1000 to sort of load all saved monitors
           .expect(200);
 
-        const found: Array<SavedObject<MonitorFields>> = apiResponse.body.monitors.filter(
-          ({ id }: SavedObject<MonitorFields>) => [id1, id2].includes(id)
+        const found: MonitorFields[] = apiResponse.body.monitors.filter(({ id }: MonitorFields) =>
+          [mon1.id, mon2.id].includes(id)
         );
-        found.sort(({ id: a }) => (a === id2 ? 1 : a === id1 ? -1 : 0));
-        const foundMonitors = found.map(({ attributes }: SavedObject<MonitorFields>) => attributes);
+        found.sort(({ id: a }) => (a === mon2.id ? 1 : a === mon1.id ? -1 : 0));
+        const foundMonitors = found.map((fields: MonitorFields) => fields);
 
         const expected = [mon1, mon2];
 
-        expect(foundMonitors).eql(expected);
+        expect(foundMonitors.map((fm) => omit(fm, 'updated_at', 'created_at'))).eql(
```

Comment:
Why are `updated_at` and `creaqted_at` omitted here?

Reply:
This is because these values are dynamic.

I added a check that they're valid dates and a comment explaining why they're omitted 949a201ef03c3507093d29db87d2a7615b4cbd99.